zenoh/src/session.rs

```diff
@@ -1272,7 +1272,7 @@ impl Session {
 
     pub(crate) fn query(
         &self,
-        selector: &Selector<'_>,
+        selector: Selector<'_>,
```

Comment:
Check if the `Selector` really needs to be passed by value or it can still be passed by reference.